x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps/install_managed_agent_step.tsx

```diff
@@ -51,6 +53,7 @@ export const InstallManagedAgentStep = ({
         enrollToken={enrollToken}
         onCopy={onCopy}
         fullCopyButton={fullCopyButton}
+        cloudFormation={cloudFormation}
```

Comment:
Can we rename this prop `cloudFormationTemplateUrl` to more accurately represent the data we're passing?